docs/src/tutorials/bank_fraud_detection_with_tfdf.ipynb

```diff
@@ -0,0 +1,1316 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "9a8e59c2-df0a-4782-8ff7-514b826f76bc",
+   "metadata": {},
+   "source": [
+    "# Detecting payment card fraud with Temporian and TensorFlow Decision Forests\n",
+    "\n",
+    "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/google/temporian/blob/main/docs/src/tutorials/bank_fraud_detection_with_tfdf.ipynb)\n",
+    "\n",
+    "Detection of fraud in online banking is critical for banks, businesses, and their consumers. The book \"[Reproducible Machine Learning for Credit Card Fraud Detection](https://fraud-detection-handbook.github.io/fraud-detection-handbook/Foreword.html)\" by Le Borgne et al. introduces the problem of payment card fraud and shows how fraud can be detected using machine learning. However, since banking transactions are sensitive and not widely available, the book uses a synthetic dataset for practical exercises.\n",
+    "\n",
+    "This notebook uses the same dataset to show how to use Temporian and [TensorFlow Decision Forests](https://www.tensorflow.org/decision_forests) to detect fraud. Temporian is used for data preprocessing and augmentation, while TensorFlow Decision Forests is used for model training. Data augmentation is often critical for temporal data, and this notebook demonstrates how complex data augmentation can be performed with ease using Temporian.\n",
+    "\n",
+    "The notebook is divided into three parts:\n",
+    "\n",
+    "1. Download the dataset and load it as a Temporian EventSet.\n",
```

Comment:
Download the dataset and import it to Temporian

Reply:
*Done